mod/da/pkg/kzg/ckzg/ckzg.go

```diff
@@ -49,11 +49,11 @@ func NewVerifier(ts *gokzg4844.JSONTrustedSetup) (*Verifier, error) {
 		len(ts.SetupG1Lagrange)*(len(ts.SetupG1Lagrange[0])-2)/2,
 	)
 	for i, g1 := range ts.SetupG1Lagrange {
-		copy(g1s[i*(len(g1)-2)/2:], hex.MustToBytes(g1))
+		copy(g1s[i*(len(g1)-2)/2:], hex.ToBytesSafe(g1))
```

Comment:
seems like coderabbit is having a valid point, what do you think @berachain/core ? It will also happen in other places where `ToBytesSafe()` is used. Are we okay with this ? I am wondering that eventually we would end up in using `ToBytes` and removing `ToBytesSafe` altogether.